src/main/java/robot/subsystems/shooter/Shooter.java

```diff
@@ -0,0 +1,48 @@
+package robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+import static robot.Constants.Shooter.*;
+import static robot.Ports.Shooter.*;
+
+public class Shooter extends Subsystem {
+    private TalonSRX shooterMaster = new TalonSRX(MASTER);
+    private VictorSPX shooterSlave = new VictorSPX(SLAVE);
```

Comment:
You can remove the word `shoot` from these variables. Also, I recommend declaring them as `final`.